skimage/util/_montage.py

```diff
@@ -31,8 +31,8 @@ def montage(arr_in, fill='mean', rescale_intensity=False, grid_shape=None,
 
     Parameters
     ----------
-    arr_in : (K, M, N[, C]) ndarray
-        An array representing an ensemble of `K` images of equal shape.
+    arr_in : (P, M, N[, C]) ndarray
+        An array representing an ensemble of `P` images of equal shape.
```

Comment:
K here is a specifically nice letter to signify that the 2D images are not meant to be interpreted as a 3D stack.